heartbeat/docker-compose

```diff
@@ -185,19 +196,25 @@ docker_compose_status()
 
 docker_compose_start()
 {
+
+	sleep 5
+
 	docker_compose_validate_all
 	docker_compose_status >/dev/null 2>&1
 	retVal=$?
 	# return success if docker service is running
 	[ $retVal -eq $OCF_SUCCESS ] && exit $OCF_SUCCESS
 
+	sleep 10
 	cd $DIR
 	$COMMAND up -d || {
 		ocf_log err "Error. docker-compose returned error $?."
+		cd /tmp
```

Comment:
You shouldnt need to run `cd /tmp` in the agent.

Reply:
fixed